shopfloor/services/checkout.py

```diff
@@ -360,7 +363,10 @@ def _domain_for_list_stock_picking(self):
         ]
 
     def _order_for_list_stock_picking(self):
-        return "scheduled_date asc, id asc"
```

Comment:
Just change this line to `priority desc, scheduled_date asc, id asc" and drop all the rest that looks overkill

Reply:
Do I also add the button to choose how to sort them, like in zone_picking? Or just leave it ordered by priority first all the time?

Reply:
I just pushed the updated version, without the button.